src/test/containers/Events/Events.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { List, ListItem, ListSubHeader} from 'react-toolbox/lib/list'
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import Events from '../../../containers/Events/Events'
+import Day from '../../../components/Day/Day'
+
+const rootEl = document.createElement('div');
```

Comment:
It should be a proper test with `it` and `expect`, you can check that render does not throw an exception for example.